ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -425,6 +434,12 @@ export default class TestDataModal extends React.Component {
           this.setState({ platform }, this.processOptions),
         title: 'Platform',
       },
+      {
+        options: tagOptions,
```

Comment:
Not all tests have tag options. Talos, the default one, hasn't integrated any of them. So an empty square-like dropdown is shown, which feels buggy.

We should rather append a modal option for tags only if there tags.